tests/filecheck/filecheckize

```diff
@@ -0,0 +1,19 @@
+# First, ignore all whitesapce-only lines
```

Comment:
👍 I posted the top-level comment before seeing this, but maybe this comment could be expanded a bit to provide the explanation why we would want to do this.